### ISSUE TITLE
Increase Menu and Popup Support for Accessibility?

### ISSUE THREAD

**Issue by BlindGuyNW** [shiiion/steinsgate_textractor#22]
Hi All,

I don't know if anyone still reads issues on this repo but figured it was worth posting. 

I've recently started getting into the SG games. I'm totally blind, that is, use a screen reading program to play. I have access to OCR, but it isn't ideal and obviously struggles with certain areas of the UI.

I was pleased to discover this program, which can output the majority of game text to the clipboard. My screen reader can read that without the many failings of scanning the screen.

I wonder how difficult it would be to add support for popups and the game menus, to make the experience a little smoother? For instance, currently selecting a save game to load is an exercise in frustration. I haven't done a lot with the phone yet, as I'm just on chapter 1, but suspect it would be similarly irritating.

Any help in making this possible would be very much appreciated.
